perfkitbenchmarker/providers/aws/aws_network.py

```diff
@@ -65,12 +78,35 @@ def __init__(self):
     self.firewall_icmp_set = set()
     self._lock = threading.Lock()
 
+  # @TODO: merge this not in rebase
```

Comment:
Still TODO?

Reply:
Wasn't sure if it should be separate from this PR. Fixed in [cd82eda] which is the last commit on this push, so I can pop it off and resubmit if it doesn't belong here.

Reply:
Given the size of this, it'd be great to resubmit this standalone (and any other pieces we can break off).